src/artikler/sok-sosialhjelp/komponenter/kommunesok/Kommunesok.tsx

```diff
@@ -71,34 +87,46 @@ const KommuneSok: React.FC<Props> = ({
                 <b>{ledetekst}</b>
             </Normaltekst>
             <br />
-            <NavAutocomplete
-                placeholder={placeholderTekst}
-                suggestions={suggestions}
-                ariaLabel={ariaLabel}
-                id="kommunesok"
-                onSelect={(suggestion: Suggestion) => onSelect(suggestion)}
-                onReset={() => onReset()}
-            />
-            {currentSuggestion && soknadTilgjengelig && (
+            {kommuneNrService.restStatus === REST_STATUS.OK &&
+            tilgjengeligeKommunerService.restStatus === REST_STATUS.OK ? (
+                <NavAutocomplete
+                    placeholder={placeholderTekst}
+                    suggestions={suggestions}
+                    ariaLabel={ariaLabel}
+                    id="kommunesok"
+                    onSelect={(suggestion: Suggestion) => onSelect(suggestion)}
+                    onReset={() => onReset()}
+                />
+            ) : (
+                <NavFrontendSpinner />
```

Comment:
Skal vi spinne ved feil også?

Reply:
👎🏻  fordi jeg stemmer nei. 
(👍🏻  fordi det var en bra kommentar!)

Reply:
Jeg syns vi litt for ofte velger å spinne (/lastestriper) når kall mot server går feil. Så denne er litt generell.

Reply:
Ja, her burde vi rydde opp. 
Lager en veldig generell sak på det her: https://jira.adeo.no/browse/DIGISOS-1951 
Så vi ikke glemmer det.

Reply:
Jeg tenker egentlig at 
"Vi fant ikke kommunen du søkte etter, sjekk at du har stavet riktig"
Er bedre enn spinner ved feil.
Da vet ihvertfall brukeren at siden ikke laster lenger og ikke venter i en evighet. 

Men beste er jo en god feilmelding, det kan tas i en annen sak

Reply:
Da tar jeg dette i en egen PR. :)